### ISSUE TITLE
Web page is not being navigated sequentially with the Tab key at Dashboard screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#824]
**Describe the bug**
Web page is not being navigated sequentially with the Tab key at Dashboard screen. The sequence of focus is unordered.

**To Reproduce**
1. Go to Dashboard screen (http://localhost:9992/cm/app/?view=dash&pathType=unknown)
2. Use Tab key to focus on different components of the screen.
3. Focus is not moving in sequence for various Dashboard Gadgets such as "Pages By Status" gadget.

**Expected behavior**
Web page should be navigated sequentially with the Tab key.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by rgarg24** [comment]
Duplicate issue. Bug already raised #793.
